### PR TITLE
EOS notice

### PR DESCRIPTION
**Pull request by jeffreyc-splunk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 >ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
 
+> # :warning: End of Support (EOS) Notice
+> **The SignalFx Smart Agent has reached End of Support.**
+>
+> The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor.
+> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
+
```
Comment (aunshc) [inline README.md:8]: LGTM, keeping this in mind I suggest we remove the reference to the deprecation notice below.

File: docs/smartagent-deprecation-notice.md [hunk 2]
```diff
@@ -1,11 +1,5 @@
...
->* Splunk products
->* Fluentd for log capture
->* Tools to support migration from SignalFx products
-
-> Please see [this](https://docs.splunk.com/observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html#nav-Migrating-from-SignalFx-Smart-Agent-to-Splunk-Distribution-of-OpenTelemetry-Collector) documentation for details to assist with moving to the Splunk Distribution of OpenTelemetry Collector.
-
->Note that the date affects the standalone agent; the Smart Agent monitors will be available and supported with the [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/README.md) in the Splunk Distribution of OpenTelemetry Collector.
+> The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor.
+> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
```
Comment (aunshc) [inline docs/smartagent-deprecation-notice.md:5]:
```suggestion
> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).

>Note that this affects the standalone agent; the Smart Agent monitors will be available and supported with the [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/README.md) in the Splunk Distribution of OpenTelemetry Collector.

```

**Comment by aunshc** [comment]
@aurbiztondo-splunk @theletterf can you review as well? Thanks!

**Review by rmfitzpatrick — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,12 +1,19 @@
 >ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
 
+> # :warning: End of Support (EOS) Notice
```
Comment (theletterf) [inline README.md:3]:
```suggestion
> # :warning: End of Support (EoS) Notice
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+> **The SignalFx Smart Agent has reached End of Support.**
+>
+> The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor.
+>
+> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
+>
+> Note that this affects the standalone agent; the Smart Agent monitors will be available and supported with the [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/README.md) in the Splunk Distribution of OpenTelemetry Collector.
```
Comment (theletterf) [inline README.md:10]:
```suggestion
> The [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector) is the successor. Smart Agent monitors are available and supported through the [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/README.md) in the Splunk Distribution of OpenTelemetry Collector.
>
> To learn how to migrate, see [Migrate from SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/smart-agent-migration-to-otel-collector.html).
```

**Pull request merged**